### PR TITLE
Doc more dbep

### PR DESCRIPTION
**Pull request by tacaswell**
This should be merged after #322 as it shares commit.

### CODE REVIEW

File: doc/source/DBEP/DBEP02.rst [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+are generated by calls to ``reg.register_*`` and the datum uids are
+subsequently returned to the *RunEngine* via ``obj.read``.  The
+proposed change is:
+
+ 1. *ophyd* objects would be responsible for generating the full *Resource*
+    and *Datum* documents and providing them to the *RunEngine* to be
+    emitted.  *ophyd* may provide some helpers to make generating compliant
+    documents easy.
+
+    a. Similar to the current documents, a *Resource* must be emitted
```
Comment (ronpandolfi) [inline doc/source/DBEP/DBEP02.rst:73]: Is this a good place to talk about how interleaved/muxed data could be emitted? We had discussed this as an issue with only having one *Start* doc 'open' at a time. Can you note any plans to accommodate such acquisition modes?

Comment (danielballan) [reply doc/source/DBEP/DBEP02.rst:73]: Looks like this comment got lost. The issue of supporting multiple RunStart documents "open" at a time is being tracked as part of https://github.com/bluesky/bluesky/issues/585

Comment (danielballan) [reply doc/source/DBEP/DBEP02.rst:73]: Looks like this comment got lost. The issue of supporting multiple RunStart documents "open" at a time is being tracked as part of bluesky/bluesky#585

**Comment by danielballan** [comment]
What would you like to do with this @tacaswell?

**Pull request closed without merging**